app/assets/javascripts/search.js

```diff
@@ -35,6 +149,7 @@ $(document).ready(function() {
     clickOnEnter(e, ".m-search-ui button.search");
   });
 
+  /******************************************************************************************************/
   /* *** setup Save as Report *** */
```

Comment:
this JS file is getting pretty long! any ideas on how we could make it more modular / break it up?